cliquet_docs/quickstart.rst

```diff
@@ -200,6 +200,33 @@ Advanced initialization
 .. autofunction:: cliquet.initialize
 
 
+
+Declare API capabilities
```

Comment:
I would put this in the "plugins" section of the docs and not in the quickstart.